system_tests/system_tests.go

```diff
@@ -355,14 +356,14 @@ CONSOLE_LOG=new`
 
 			It("talks amqps with RabbitMQ", func() {
 				var err error
-				username, password, err = getRabbitmqUsernameAndPassword(clientSet, "rabbitmq-system", "tls-test-rabbit")
+				username, password, err = getUsernameAndPassword(ctx, clientSet, "rabbitmq-system", "tls-test-rabbit")
 				Expect(err).NotTo(HaveOccurred())
 
 				// try to publish and consume a message on a amqps url
 				sentMessage := "Hello Rabbitmq!"
-				Expect(rabbitmqAMQPSPublishToNewQueue(sentMessage, username, password, hostname, amqpsNodePort, caFilePath)).To(Succeed())
+				Expect(publishToNewQueueAMQPS(sentMessage, username, password, hostname, amqpsNodePort, caFilePath)).To(Succeed())
```

Comment:
Same as above `publishToQueueViaAMQPS`?

Reply:
Updated to `publishToQueueAMQPS`. Thanks!